backend/api/models.py

```diff
@@ -37,6 +37,7 @@ def __str__(self):
 class Content(models.Model):
     content_id = models.AutoField(primary_key=True)
     japanese_slang = models.CharField(max_length=255)
+    english_slang = models.CharField(max_length=255)
```

Comment:
Maybe is better a textfield because can contain long string

Reply:
@FilippoQuattrocchi 
I extended each field to 1024, not only english_slang.